source/features/bypass-checks.tsx

```diff
@@ -9,7 +9,8 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 		detailsLink.href,
 		'[data-hydro-click*="check_suite.external_click"]'
 	);
-	detailsLink.href = directLink!.href;
+
+	detailsLink.href = directLink?.href ?? detailsLink.href;
```

Comment:
```suggestion
	if (directLink) {
		detailsLink.href = directLink.href;
	}
```

BUT this just silences the error.

Check again. The page where this fails is this: https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=788471140

And you can see that **there is** an external link on [that](https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=788471140) page:

<img width="900" alt="" src="https://user-images.githubusercontent.com/1402241/85147882-beb64c00-b24f-11ea-8a7d-c2d4f709023a.png">

Reply:
Not resolved

Reply:
Not resolved. This condition should not be necessary.

> Check again. The page where this fails is this: https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=788471140
> 
> And you can see that **there is** an external link on [that](https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=788471140) page

Reply:
I linked the wrong page
https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=789424924

Reply:
It's not a link issue.

1. Visit https://github.com/twbs/bootstrap/pull/31090
2. See `bypass-checks` fail in the console

![sad](https://user-images.githubusercontent.com/1402241/85232937-6a53cd80-b403-11ea-9949-a7f1a35d9c39.gif)

That page is the one I linked

Reply:
Right and that page **does not have a href**

Reply:
I blame Firefox. Sometimes it shows me `https://github.com/twbs/bootstrap/pull/31090/checks?check_run_id=792572073` and I visit it and see the link, but then it crashes skipping that breakpoint making me think _that's_ the link crashing. Apologies

Reply:
NP